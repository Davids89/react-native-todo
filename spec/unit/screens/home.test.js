import React from 'react'
import {render} from '@testing-library/react-native'
import Home from 'screens/home'

test('render correctly', () => {
  const {toJSON} = render(<Home />)

  expect(toJSON()).toMatchSnapshot()
})