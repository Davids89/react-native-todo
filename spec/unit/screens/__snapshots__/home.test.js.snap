// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<View>
  <Text>
    Home Screen
  </Text>
</View>
`;
